packages/aws-rfdk/lib/core/lib/mountable-efs.ts

```diff
@@ -86,8 +126,14 @@ export class MountableEfs implements IMountableLinuxFilesystem {
 
     const mountDir: string = path.posix.normalize(mount.location);
     const mountOptions: string[] = [ MountPermissionsHelper.toLinuxMountOption(mount.permissions) ];
+    if (this.props.accessPoint) {
+      mountOptions.push(
```

Comment:
Does passing these to the nfsv4 mount helper cause a break?

Minimally, there should be a warning on the MountableEfs's access point property says that the option is only available when mounting onto instances with the EFS mount helper available (e.g. Amazon Linux 2). Is the EFS helper available in any other distro's package repo?

Reply:
Addition -- Found: https://docs.aws.amazon.com/efs/latest/ug/installing-amazon-efs-utils.html#installing-other-distro

Access points for other distros can be a followup if it's asked for, imo. Restricting to AL2 only for now seems reasonable.

Reply:
Good point here. We are unable to detect the target Linux distro at synthesis time. One idea is we could add an optional boolean to indicate AL2. If omitted or false, we could raise the warning. Using it would look like:

```ts
new MountableEfs(..., {
  
}).mountToLinuxInstance(host, {
  isAmazonLinux2: true,
});
```

We could use this API to dismiss CDK warnings when not known and hard-fail at synthesis when known and false. What do you think?

Reply:
Discussed offline... decided not to do something like the `isAmazonLinux2` property.